sdks/python/apache_beam/utils/urns.py

```diff
@@ -26,11 +26,11 @@
 from apache_beam.utils import proto_utils
 
 
-PICKLED_WINDOW_FN = "beam:window_fn:pickled_python:v0.1"
-GLOBAL_WINDOWS_FN = "beam:window_fn:global_windows:v0.1"
-FIXED_WINDOWS_FN = "beam:window_fn:fixed_windows:v0.1"
-SLIDING_WINDOWS_FN = "beam:window_fn:sliding_windows:v0.1"
-SESSION_WINDOWS_FN = "beam:window_fn:session_windows:v0.1"
+PICKLED_WINDOW_FN = "beam:windowfn:pickled_python:v0.1"
+GLOBAL_WINDOWS_FN = "beam:windowfn:global_windows:v0.1"
```

Comment:
We could have a script that goes through the proto file (or something else) and pulls the name out and generates a .java, .py, .go, ... file of constants. Perhaps overkill though. Ideally this'd be co-located with the semantic definition of the urn as well, if needed.